pkg/apis/apis/v1alpha1/permissionclaims/labels.go

```diff
@@ -39,6 +39,20 @@ func ToLabelKeyAndValue(exportClusterName logicalcluster.Name, exportName string
 	return apisv1alpha1.APIExportPermissionClaimLabelPrefix + exportHash, claimHash, nil
 }
 
+// ToReflexiveAPIBindingLabelKeyAndValue returns label key and value that is set (as fallback for filtering)
+// on APIBindings that point to the given APIExport and the binding has not accepted a claim to it.
```

Comment:
not sure if I understand `and the binding has not accepted a claim to it`. Does it mean it will/should be deleted once the claim has been accepted?